biblib/views/http_errors.py

```diff
@@ -53,7 +53,7 @@
 )
 SOLR_RESPONSE_MISMATCH_ERROR = dict(
     body='Solr response does not contain the same number of bibcodes as the '
-         'request.',
+         'request, or only the raw library was requested.',
```

Comment:
Is this text update needed? As far as I can tell, this error is only used within a `if not raw_library` block